src/commonpp/thread/detail/logger.cpp

```diff
@@ -0,0 +1,22 @@
+/*
+* File: src/commonpp/thread/detail/logger.hpp
+* Part of commonpp.
+*
+* Distributed under the 2-clause BSD licence (See LICENCE.TXT file at the
+* project root).
+*
+* Copyright (c) 2016 Thomas Sanchez.  All rights reserved.
+*
+*/
+
+#include "logger.hpp"
+
+namespace commonpp
+{
+namespace thread
+{
+
+DECLARE_LOGGER(thread_logger, "commonpp::thread");
```

Comment:
Not 100% sure whether it should be `DECLARE_LOGGER` or `CREATE_LOGGER` here. At least this patch fixed a linker error I received in httpp complaining about duplicate `thread_logger` symbol.